flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/model/SourceBuilder.java

```diff
@@ -55,8 +55,8 @@ public class SourceBuilder {
 
     private TokenBuilder tokenBuilder;
     public final FlowNode flowNode;
-    private final WorkspaceManager workspaceManager;
-    private final Path filePath;
+    public final WorkspaceManager workspaceManager;
+    public final Path filePath;
```

Comment:
Is this a necessary change?

Reply:
Yes, need to access these fields outside from the class.